PentaStagione.Domain/PizzaIngredient.cs

```diff
@@ -1,10 +1,17 @@
-﻿using PentaStagione.Infrastructure.Domain;
+﻿using System;
+using PentaStagione.Infrastructure.Domain;
 
 namespace PentaStagione.Domain
 {
     public class PizzaIngredient : IEntity
     {
-        //TODO: generate id
+        public PizzaIngredient()
+        {
+            Id = Guid.NewGuid().ToString();
```

Comment:
Pizza ingredients can be a lot in the system. Is it a good idea to have this Id as string? If not, please change the type to Guid